Classes/TsConfig/Loader.php

```diff
@@ -24,6 +23,20 @@
  */
 class Loader
 {
+
+    /**
+     * @var PackageHelper
+     */
+    protected $packageHelper = null;
```

Comment:
I would get rid of the default `= null` assignment as the property is null as long it is not set anyway.